bin/i18n/sync-in.rb

```diff
@@ -36,6 +37,30 @@ def sync_in
   raise e
 end
 
+# This function localizes all content in studio.code.org/docs
+def localize_docs
+  puts "Preparing docs content"
+  docs_content_file = File.join(I18N_SOURCE_DIR, "docs", "programming_environments.json")
+  programming_env_docs = {}
+  ProgrammingEnvironment.all.each do |env|
+    name = env.name
+    programming_env_docs[name] = {
+      'name' => env.properties["title"],
+      'description' => env.properties["description"],
+      'categories' => {},
+    }
+    env.categories_for_navigation.each do |category|
```

Comment:
some documentation on how this structure will look at the end would be super helpful!